torch/ao/quantization/fx/_decomposed.py

```diff
@@ -1185,3 +1185,22 @@ def fake_quant_per_channel_meta(
     quant_max: int,
 ) -> torch.Tensor:
     return torch.empty_like(input)
+
+
+quantized_decomposed_lib.define(
+    "convert_element_type.no_fuse(Tensor input, ScalarType dtype) -> Tensor"
```

Comment:
do you need to add this op to https://github.com/pytorch/pytorch/blob/fd553b9817cb145e3cfaa52a08a831b9edf2fec7/torch/_inductor/constant_folding.py#L132 so it's not constant folded?

Reply:
I have added this in the next PR. Thanks